cawemo-to-web-modeler-migration/migrate-cawemo-to-web-modeler.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/sh 
+
```

Comment:
I would usually set these options at the beginning to make the script fail early in case of errors:
`set -eu -o pipefail`

For example, when I tried to run it, `jq` was not found first, but the "MIGRATION SUCCESS" message was displayed anyway.